File: src/main/java/io/github/svegon/capi/mixininterface/IClientWorld.java
package io.github.svegon.capi.mixininterface;

import net.minecraft.client.network.ClientPlayNetworkHandler;
import net.minecraft.client.network.PendingUpdateManager;

public interface IClientWorld {
    ClientPlayNetworkHandler getNetworkHandler();

    PendingUpdateManager pendingUpdateManager();
}

File: src/client/kotlin/io/github/svegon/mclientapi/client/mixinterface/IClientWorld.kt
package io.github.svegon.mclientapi.client.mixinterface

import net.minecraft.client.Minecraft
import net.minecraft.client.multiplayer.ClientPacketListener

interface IClientWorld {
    val `mClientAPI$minecraft`: Minecraft

    val `mClientAPI$packetListener`: ClientPacketListener
}
